### DISCUSSION TITLE
Timed Deletion of Media

### DISCUSSION THREAD

**Discussion by randomjeep** [sct/overseerr#2888] [Feature Requests]
It would be great if you could set a time period for certain users downloads. ex. keeping a movie from x user or user group for 2 weeks before being deleted.

seeing as overseerr connects to the arrs unmonitoring and deleting media after x amount of time would be amazing.

**Comment by TheCatLady** [comment]
Overseerr will never handle deletion of media. Please see #1924.

**Reply by randomjeep** [reply]
I thought I saw a remove media button but its actually clear data. Oops!
